saleor/warehouse/tasks.py

```diff
@@ -11,3 +11,10 @@ def delete_empty_allocations_task():
     count, _ = Allocation.objects.filter(quantity_allocated=0).delete()
     if count:
         task_logger.debug("Removed %s allocations", count)
+
+
+@app.task
+def update_stocks_quantity_allocated_task():
+    for stock in Stock.objects.iterator():
+        stock.recalculate_quantity_allocated()
+    task_logger.debug("Updated quantity_allocated on all stocks.")
```

Comment:
We should log for which stocks we need to update `quantity_allocated` and we should log a diffrence. 
```suggestion
    task_logger.info("Updated quantity_allocated on all stocks.")
```